pkg/services/user/userimpl/store.go

```diff
@@ -79,7 +78,7 @@ func (ss *sqlStore) Insert(ctx context.Context, cmd *user.User) (int64, error) {
 	if err != nil {
 		return 0, err
 	}
-	return userID, nil
+	return cmd.ID, nil
```

Comment:
I have mixed feelings for this change; I would prefer not to rely on the underline library for accessing the assigned primary key

Reply:
Sure, but where else are we getting that value from? The returned int from `Insert` is the number of rows affected, not the ID - I found that bug a couple of places.

Reply:
for posterity, I ended up adding GetByID using the cmd.ID to double check that the user was created and/or the ID was updated accurately.